client.go

```diff
@@ -35,6 +35,8 @@ type Client interface {
 	// topic/partition, as determined by querying the cluster metadata.
 	Leader(topic string, partitionID int32) (*Broker, error)
 
+	Any() *Broker
```

Comment:
I don't think there's a good reason to expose this publicly, it's never been a well-defined method in the first place.

Reply:
I need to call broker.Createtopic, broker.Deletetopic etc from admin.go . But I need a broker that is open so I thought of exposing this and accessing a broker which is open. Is there a way to access a broker which is open easily or we need to do that in this file. Which do you recommend ?

Reply:
Will use client.Brokers instead of any.